inginious/frontend/app.py

```diff
@@ -106,6 +108,30 @@ def _put_configuration_defaults(config):
         config['allowed_file_extensions'] = [".c", ".cpp", ".java", ".oz", ".zip", ".tar.gz", ".tar.bz2", ".txt"]
     if 'max_file_size' not in config:
         config['max_file_size'] = 1024 * 1024
+
+    if 'session_parameters' not in config or 'secret_key' not in config['session_parameters']:
+        print("/!\\ WARNING /!\\", file=sys.stderr)
+        print("You did not define the 'session_parameters' entry in your INGInious configuration file.", file=sys.stderr)
+        print("A default secret key has been used. This is potentially dangerous.", file=sys.stderr)
```

Comment:
I think we should simply not allow to run the app without the secret key set in the configuration file instead of using a default one. Unless my memory fails, I think this is the Flask behaviour.